### PR TITLE
Touch bug 1417530: Add more fixtures

### PR DESCRIPTION
**Pull request by phlax**
this will also ease the landing of #766

### CODE REVIEW

File: tests/fixtures/site.py [hunk 1]
```diff
@@ -1,41 +1,150 @@
...
 import pytest
 
+from django.utils import timezone
 
-@pytest.fixture
-def site_matrix(factories):
+
+def _add_project_resources(project):
+    return [
+        dict(project=project, total_strings=(i + 5) ** 3)
```
Comment (adngdb) [inline tests/fixtures/site.py]: All those numbers in this file look like magic numbers to me. Can you add a comment explaining what they correspond to? Like, why `+ 5` and why `** 3`?

File: tests/fixtures/site.py [hunk 2]
```diff
@@ -1,41 +1,150 @@
...
+def _add_translated_resources(locales, project, resources):
+    _kwargs = []
+    for i, resource in enumerate(resources):
+        for locale in locales:
+            _kwargs.append(
+                dict(locale=locale,
+                     approved_strings=(i + 2) ** 3,
+                     translated_strings=(i + 1) ** 3,
+                     fuzzy_strings=i ** 3))
+    return _kwargs
```
Comment (adngdb) [inline tests/fixtures/site.py]: These three functions are called `_add_something` but I don't see them adding anything. Shouldn't they be called `_get_something` instead, as they return the items you're interested in?

File: tests/fixtures/site.py [hunk 3]
```diff
@@ -1,41 +1,150 @@
...
+    for kwargs in translated_resource_kwargs:
+        kwargs["latest_translation"] = _get_latest_translation(latest, kwargs)
+    translated_resources = (
+        factories['translated_resource'](
+            batch_kwargs=translated_resource_kwargs))
+    return translated_resources
+
+
+@pytest.fixture
+def site_matrix(factories):
```
Comment (adngdb) [inline tests/fixtures/site.py]: This file could do with a lot more docstrings on functions, especially this one and `_create_site_root`. I'm not exactly sure what you mean by "site matrix" and "site root", so a short explanation could be quite useful. :)

File: tests/fixtures/site.py (hunk 2 above)
Comment (phlax) [reply tests/fixtures/site.py]: yep, renamed

File: tests/fixtures/site.py (hunk 3 above)
Comment (phlax) [reply tests/fixtures/site.py]: added, hope its enough to make this code a bit more readable

**Pull request merged**